inception-project/inception
# #807 - IllegalArgumentException when creating a Statement

**What's in the PR**
* Fix bug that occurs when a new statement is created
* don't redraw statementListWrapper when creating a new statementgroup
* move methods for creating a new statementgroup to NewStatementGroupFragment
* Closes #807

**How to test manually**
* go to KBPage, create new statements.
* check that statements are created properly and that no exception is thrown

**Automatic testing**

**Documentation**
